src/components/plots/xyPlot.ts

```diff
@@ -71,6 +71,49 @@ module Plottable {
       return this;
     }
 
+    protected _updateExtentsForAttr(attr: string) {
+      if (attr === "x") {
+        return super._updateExtentsForAttr(attr, this._xFilter());
+      } else if (attr === "y") {
+        return super._updateExtentsForAttr(attr, this._yFilter());
+      }
+      return super._updateExtentsForAttr(attr);
+    }
```

Comment:
Would a better design be... to have `_updateExtentsForAttr` to expose an endpoint that this can override?  Now we are putting the idea of filtering on the super class when it doesn't seem like it should exist there